### DISCUSSION TITLE
Double/half the width/height of the window

### DISCUSSION THREAD

**Discussion by isaacy2012** [rxhanson/Rectangle#1369] [Ideas]
I use multiple monitors of equal size, and sometimes I want a window to span both displays. 
I often have the window on the left side and want to make it span both screens, and then return back to before. 

This has been discussed before but not implemented: https://github.com/rxhanson/Rectangle/discussions/984, but I think a doubling/halving implementation is much easier than the complexity of ascertaining which monitors to combine.

Happy to implement and make a PR for this feature.

**Comment by rxhanson** [comment]
Thanks for looking to contribute! This makes sense to me, and I'm good with merging a good PR for it. 

My preferred way of handling additional window action contributions is that they are not added to the UI at first. There are a number of examples of this referenced in the terminal commands readme: https://github.com/rxhanson/Rectangle/blob/main/TerminalCommands.md#add-an-extra-centering-command-with-custom-size

Let me know if you need anything else to get started.
